packages/plugin-styled-components/src/index.ts

```diff
@@ -57,39 +59,43 @@ export const pluginStyledComponents = (
       });
     };
 
+    // use modifyRsbuildConfig to get useSSR default value (can be override by pluginOptions)
```

Comment:
From my perspective, we should not use the `modifyRsbuildConfig` to `get` config.

If we want to get the config, we can use `api.getRsbuildConfig()`.

Reply:
Calling `api.getRsbuildConfig()` in modifyEnvironmentConfig hook is currently not supported because RsbuildConfig contains EnvironmentConfig, which has not yet been determined.

Reply:
No, it is allowed call `api.getRsbuildConfig()` anywhere and it should alway return the current (latest) config.

For example, if `modifyRsbuildConfig` has been called, `api.getRsbuildConfig()` should return the normalized Rsbuild config.

![image](https://github.com/user-attachments/assets/0b287a03-11e7-4583-975a-754398a5ecc2)

Reply:
get it!